Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -249,7 +249,6 @@ def get_event_status(args: Dict[str, Any], client: Client, default_job_id) -> Co
     headers = ["event_id", "ingest_timestamp", "status", "error_message",
                "bucket_state", "result", "confidence_score", "outlier", "arcanna_label"]
     return_data = CommandResults(
-        outputs_prefix='Arcanna.Event',
```

Comment:
Why would you do that while all the context outputs in this command start with this prefix?

Reply:
because I can call it multiple times and put it in a nested way,

Reply:
Maybe you can just remove the `'Arcanna.Event'` from the outputs object (line 247) and leave the outputs_prefix and add the `outputs_key_field` to select the field on which output to override by.
See https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults and https://xsoar.pan.dev/docs/integrations/context-and-outputs

Also, please take into account that every change in the output may break backwards for other customers that use this and we'll need BC release notes for this.

Reply:
basically I need to fully override it every time I call this function.
that's why I've removed it.
can we setup a call to talk over ?

Reply:
Yes, DM you on Slack DFIR